src/template/ar/careers.ejs

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="ar" translate="no">
+
+<head>
+    <title>وظائف | Martspec</title>
+    <meta name="description"
+        content="فريقنا عبارة عن مجموعة من المهنيين الشباب الواعدين الذين يستمتعون بأداء عملهم. نحن نبحث عن الأشخاص الموهوبين والطموحين والمستعدين للنمو والتطور معنا.">
+
+    <!-- Language Support -->
+    <link rel="alternate" hreflang="x-default" href="https://martspec.com/ar/careers" />
+    <link rel="alternate" hreflang="en" href="https://martspec.com/careers" />
```

Comment:
We do not need this link because we have /en later that points to the same page and those are canonical links. In all templates you can just delete line with language code in URL.

Reply:
Right?

```html
    <!-- Language Support -->
    <link rel="canonical" href="https://martspec.com/ar/careers">
    <link rel="alternate" hreflang="x-default" href="https://martspec.com/ar/careers" />
    <link rel="alternate" hreflang="ar" href="https://martspec.com/ar/careers" />
    <link rel="alternate" hreflang="de" href="https://martspec.com/de/careers" />
    <link rel="alternate" hreflang="en" href="https://martspec.com/en/careers" />
    <link rel="alternate" hreflang="es" href="https://martspec.com/es/careers" />
    <link rel="alternate" hreflang="fr" href="https://martspec.com/fr/careers" />
    <link rel="alternate" hreflang="hi" href="https://martspec.com/hi/careers" />
    <link rel="alternate" hreflang="ja" href="https://martspec.com/ja/careers" />
    <link rel="alternate" hreflang="pt" href="https://martspec.com/pt/careers" />
    <link rel="alternate" hreflang="ru" href="https://martspec.com/ru/careers" />
    <link rel="alternate" hreflang="uk" href="https://martspec.com/uk/careers" />
    <link rel="alternate" hreflang="zh" href="https://martspec.com/zh/careers" />
```